### PR TITLE
AUI-2097 Close toolbar before add it to another field

### PR DESCRIPTION
**Pull request by fernandosouza**

### CODE REVIEW

**Comment by thiago-rocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by thiago-rocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/aui-form-builder/js/aui-form-builder-layout-builder.js [hunk 1]
```diff
@@ -210,12 +212,20 @@ A.FormBuilderLayoutBuilder.prototype = {
...
 
+        this._selectFirstValidMoveTarget();
+
         this._cancelMoveFieldHandles = [
             A.one(A.config.doc).on('click', A.bind(this._onClickOutsideMoveTarget, this)),
             A.one(A.config.doc).on('key', A.bind(this._onEscKeyPressMoveTarget, this), 'down:27')
         ];
     },
 
+    _selectFirstValidMoveTarget: function() {
```
Comment (thiago-rocha) [inline src/aui-form-builder/js/aui-form-builder-layout-builder.js]: Can you fix the alphabetical order here? This should go before _setMoveButtonData

File: src/aui-form-builder/js/aui-form-builder-layout-builder.js [hunk 2]
```diff
@@ -210,12 +212,20 @@ A.FormBuilderLayoutBuilder.prototype = {
 
         layout.normalizeColsHeight(layout.get('node').all('.row'));
 
+        this._selectFirstValidMoveTarget();
+
         this._cancelMoveFieldHandles = [
             A.one(A.config.doc).on('click', A.bind(this._onClickOutsideMoveTarget, this)),
             A.one(A.config.doc).on('key', A.bind(this._onEscKeyPressMoveTarget, this), 'down:27')
         ];
     },
```
Comment (thiago-rocha) [inline src/aui-form-builder/js/aui-form-builder-layout-builder.js]: Missing documentation here.

Comment (fernandosouza) [inline src/aui-form-builder/js/aui-form-builder-layout-builder.js]: Done.

File: src/aui-form-builder/js/aui-form-builder-layout-builder.js (hunk 1 above)
Comment (fernandosouza) [inline src/aui-form-builder/js/aui-form-builder-layout-builder.js]: Done.

**Comment by thiago-rocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/aui-form-builder/js/aui-form-builder.js [hunk 3]
```diff
@@ -532,6 +536,7 @@ A.FormBuilder = A.Base.create('form-builder', A.Widget, [
 
         if (fieldContainer) {
             if (!fieldContainer.contains(this._fieldToolbar._toolbar)) {
+                this._fieldToolbar._toolbar.removeClass('open');
```
Comment (thiago-rocha) [inline src/aui-form-builder/js/aui-form-builder.js]: Instead of use the private attribute (`_toolbar`) use the close function.

Comment (fernandosouza) [inline src/aui-form-builder/js/aui-form-builder.js]: Oh! My mistake. Done.

**Pull request merged**